docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.8'
+services:
+  app:
+    build: .
+    container_name: amigos_ecom_app_c
+    image: app_amigos_ecom
+    ports:
+      - 3000:3000
+    environment:
+      - DB_NAME=amigos_ecom
+      - DB_USERNAME=postgres
+      - DB_USER_PASSWORD=test1234
+      - DB_HOST=db
+    depends_on:
+      - db
+
```

Comment:
find a way to add volumes tag on app services so that editing in dev mode can be synced in real time with changes in docker